src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/taskmanagementservicetests/CompleteTaskTest.java

```diff
@@ -331,41 +271,6 @@ void completeTask_should_throw_role_assignment_verification_exception_when_has_a
         verify(camundaService, times(0)).completeTask(any(), anyBoolean());
     }
 
-    @Test
-    void completeTask_should_throw_task_state_incorrect_exception_when_task_has_no_assignee_gp_flag_is_off() {
```

Comment:
regardless of gp flag, this edge case test should behave the same way. i.e. throw task state incorrect exception. Should this be removed?